06_gpu_and_ml/stable_lm/main.py

```diff
@@ -0,0 +1,146 @@
+import time
+from pathlib import Path
+
+import torch
+from huggingface_hub import snapshot_download
```

Comment:
Can inline this import inside `compile_models` so it's not needed locally.

Reply:
Agreed. I just didn't want to re-run the image build at the first pass.

Reply:
Agreed on minimizing local imports